src/utils.js

```diff
@@ -1,22 +1,44 @@
 const CALLBACK_NAME = '_grecaptchaonloadcallback'
 
+let loadScriptPromise
+
+// Returns a promise that resolves once Google's reCAPTCHA library is loaded. If the library is already loaded, then no
+// work is performed, otherwise the library is dynamically loaded.
+// @param {string} [locale] - one of the language codes at https://developers.google.com/recaptcha/docs/language.
```

Comment:
Could you remove these jsdoc comments?

Reply:
Because the comments are `//` instead of `/** */`, they aren't picked up by the JSDoc processor. So they're not really JSDoc comments and they don't show up in the README's API section. However, they look like JSDoc comments which could be confusing, so I understand why you are requesting this change.

But I want to fully understand your request. Do you want the comments gone entirely, or just reformatted to remove the `@param` and `@returns` JSDoc keywords?

Reply:
Just remove the `@param` and `@returns` comments (that look like JSDochere, it doesn't add too much value here.

I generally don't use comments at all, just in rare cases I think that they are useful. In the case here, your comments add more value to the code.